docs/pages/access-controls/sso/azuread.mdx

```diff
@@ -204,11 +206,11 @@ should enable token encryption, read the Azure AD
 
 ### Set up Teleport token encryption
 
-Start with generating a public/private key and a certificate. You will set up the public
-certificate with Azure AD and the private key with Teleport.
+Start with generating a public/private key and a certificate. You will set up
```

Comment:
I'd suggest something like this (except with the real information rather than my fiction)...

Token encryption requires a public/private key and a certification. The public key and certificate are used by Azure AD to authenticate users with the private key held by the Teleport client. 

To set up Teleport token encryption:

1. Generate a public/private key pair and a certificate by running the following command:

Reply:
This is existing content that only shows as changed because I formatted it. Honestly, I didn't do this as part of my testing so I can't fill in the "real information rather than my fiction" part of this suggestion.